src/modules/sdai/LiveSDAIBalance.tsx

```diff
@@ -84,3 +78,19 @@ function useRefresh(ms: number) {
     };
   }, []);
 }
+
+const getVisibleDecimals = (daiBalance: BigNumber | undefined): number => {
+  const changeRatio = 1.55e-9;
+  const value = daiBalance?.multipliedBy(changeRatio);
+  if (!value) {
+    return 0;
+  }
+
+  const e = value.e;
+
+  if (!e || e >= 0) return 0;
+
+  const decimals = -e;
+
+  return decimals === 1 ? 2 : decimals;
```

Comment:
can we actually add max here? just as a precuation? maybe just the previously used value ie. `7`